### PR TITLE
Add group comments in generated sys

### PR DESCRIPTION
**Pull request by EPashkin**

### CODE REVIEW

File: src/codegen/sys/lib_.rs [hunk 1]
```diff
@@ -97,21 +97,28 @@ where Type: MaybeRef<T> {
 
 fn generate_aliases(w: &mut Write, env: &Env, items: &[&Alias])
         -> Result<()> {
-    try!(writeln!(w, ""));
+    if !items.is_empty() {
+        try!(writeln!(w, "// Alases"));
```
Comment (GuillaumeGomez) [inline src/codegen/sys/lib_.rs]: Aliases

File: src/codegen/sys/lib_.rs [hunk 2]
```diff
@@ -255,19 +275,26 @@ fn generate_classes_structs(w: &mut Write, env: &Env, classes: &[&Class]) -> Res
...
 }
 
 fn generate_records(w: &mut Write, env: &Env, records: &[&Record]) -> Result<()> {
-    try!(writeln!(w, ""));
+    if !records.is_empty() {
+        try!(writeln!(w, "// Records"));
+    }
     for record in records {
         let (lines, commented) = generate_fields(env, &record.name, &record.fields);
-
```
Comment (GuillaumeGomez) [inline src/codegen/sys/lib_.rs]: Good catch! :D

**Comment by EPashkin** [comment]
Updated

**Comment by GuillaumeGomez** [comment]
All good for me. I'll let you merge if you don't mind since I'm not officially a reviewer/maintainer in here.

**Pull request merged**